models/classes/runner/QtiRunnerService.php

```diff
@@ -846,19 +794,11 @@ public function parsesItemResponse(RunnerServiceContext $context, $itemRef, $res
      * @param RunnerServiceContext $context
      * @param $responses
      * @return mixed
-     * @throws \common_Exception
+     * @throws common_Exception
      */
     public function emptyResponse(RunnerServiceContext $context, $responses)
     {
-        if (!$context instanceof QtiRunnerServiceContext) {
-            throw new InvalidArgumentTypeException(
-                'QtiRunnerService',
-                'storeItemResponse',
-                0,
-                QtiRunnerServiceContext::class,
-                $context
-            );
-        }
+        $this->assertIsQtiRunnerServiceContext($context, 'emptyResponse');
```

Comment:
The same as here https://github.com/oat-sa/extension-tao-testqti/pull/2383/files#r1221401588